src/styles/calendar.scss

```diff
@@ -0,0 +1,50 @@
+$square-size: 100px;
+$day-of-week: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+
+.calendar {
+  width: 710px;
```

Comment:
Will be better to move the with to separate variable, for not it's some magic number